### DISCUSSION TITLE
How do I submit an update for Telegram.TelegramDesktop in this case?

### DISCUSSION THREAD

**Discussion by sharunkumar** [microsoft/winget-create#420] [Q&A]
My command & output:

```ps1
➜ wingetcreate update --urls https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup-x64.4.8.10.exe https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup.4.8.10.exe https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable-x64.4.8.10.zip https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable.4.8.10.zip --version 4.8.10 Telegram.TelegramDesktop
Retrieving latest manifest for Telegram.TelegramDesktop
Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup-x64.4.8.10.exe...
Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup.4.8.10.exe...
Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable-x64.4.8.10.zip...
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Microsoft.WingetCreateCLI.Commands.UpdateCommand.<>c.<UpdateManifestsAutonomously>b__38_4(Installer i)
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Collections.Generic.HashSet`1.UnionWith(IEnumerable`1 other)
   at System.Collections.Generic.HashSet`1..ctor(IEnumerable`1 collection, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.DistinctIterator`1.ToList()
   at Microsoft.WingetCreateCLI.Commands.UpdateCommand.UpdateManifestsAutonomously(Manifests manifests)
   at Microsoft.WingetCreateCLI.Commands.UpdateCommand.ExecuteManifestUpdate(List`1 latestManifestContent, CommandExecutedEvent commandEvent)
   at Microsoft.WingetCreateCLI.Commands.UpdateCommand.Execute()
   at Microsoft.WingetCreateCLI.Program.Main(String[] args)
```

**Comment by mdanish-kh** [comment]
https://github.com/microsoft/winget-create/issues/421#issuecomment-1665039160

**Comment by Ingkoun** [comment]
> My command & output:
> 
> ```ps1
> ➜ wingetcreate update --urls https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup-x64.4.8.10.exe https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup.4.8.10.exe https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable-x64.4.8.10.zip https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable.4.8.10.zip --version 4.8.10 Telegram.TelegramDesktop
> Retrieving latest manifest for Telegram.TelegramDesktop
> Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup-x64.4.8.10.exe...
> Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tsetup.4.8.10.exe...
> Downloading and parsing: https://github.com/telegramdesktop/tdesktop/releases/download/v4.8.10/tportable-x64.4.8.10.zip...
> System.ArgumentNullException: Value cannot be null. (Parameter 'source')
>    at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
>    at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
>    at Microsoft.WingetCreateCLI.Commands.UpdateCommand.<>c.<UpdateManifestsAutonomously>b__38_4(Installer i)
>    at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
>    at System.Collections.Generic.HashSet`1.UnionWith(IEnumerable`1 other)
>    at System.Collections.Generic.HashSet`1..ctor(IEnumerable`1 collection, IEqualityComparer`1 comparer)
>    at System.Linq.Enumerable.DistinctIterator`1.ToList()
>    at Microsoft.WingetCreateCLI.Commands.UpdateCommand.UpdateManifestsAutonomously(Manifests manifests)
>    at Microsoft.WingetCreateCLI.Commands.UpdateCommand.ExecuteManifestUpdate(List`1 latestManifestContent, CommandExecutedEvent commandEvent)
>    at Microsoft.WingetCreateCLI.Commands.UpdateCommand.Execute()
>    at Microsoft.WingetCreateCLI.Progra__m.Main(String[] args)[]()
> ```

****@

**Reply by sharunkumar** [reply]
🤔
